recipe/meta.yaml

```
@@ -1,13 +1,13 @@
 {% set name = "fastparquet" %}
-{% set version = "2024.2.0" %}
+{% set version = "2024.5.0" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/f/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 81a8f60c51793eb2436b4fdbbf115ff8578a4a457a179240bc08f9d9573d57a4
+  sha256: dffd1d0ac6e89e31c5b6dacf67a8d299d4afbbcf0bf8b797373904c819c48f51
 
 build:
   skip: true  # [py<38]
```

Looks like Python 3.8 was [dropped in `fastparquet` version `2024.5.0`]( https://github.com/dask/fastparquet/commit/f020179c2684f4394a09791ce2d7e171c7bb010d#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7 ). Hence Python 3.8 CI jobs fail

So think we need to update this `skip` accordingly and [re-render]( https://conda-forge.org/docs/maintainer/infrastructure/#conda-forge-admin-please-rerender )

```suggestion
  skip: true  # [py<39]
```